apps/finance/app/src/components/IdentityManager/IdentityManager.js

```diff
@@ -9,6 +9,38 @@ const IdentityContext = React.createContext({
     Promise.reject(Error('Please set resolve using IdentityProvider')),
 })
 
+export function useIdentity(address) {
```

Comment:
Let's move this down to the end of the file, and export it in the same `export` declaration as the other components here.